src/app/public/modules/repeater/repeater-adapter.service.ts

```diff
@@ -11,7 +16,29 @@ import {
 export class SkyRepeaterAdapterService {
   private host: ElementRef;
 
-  constructor(private repeaterService: SkyRepeaterService) {}
+  constructor(
+    private repeaterService: SkyRepeaterService,
+    private skyAdapterService: SkyCoreAdapterService
+  ) {}
+
+  public getFocusableChildren(element: HTMLElement, options: SkyFocusableChildrenOptions): HTMLElement[] {
+    return this.skyAdapterService.getFocusableChildren(element, options);
+  }
+
+  public setTabIndexOfFocusableElems(
```

Comment:
I would avoid abbreviations in property and method names, if possible. This will eliminate user confusion.

Reply:
Done.